### PR TITLE
feat: add unassigned filter for assign users list

### PR DESCRIPTION
**Pull request by richard015ar**
Issue: https://github.com/pressbooks/pressbooks-multi-institution/issues/104

This PR:
- Centralized books/assign.blade.php and users/assign.blade.php templates.
- Adds the Unassigned link filter in Assign Users and Assign Books pages.

### Testing case
- As a Super Admin or NM go to Institutions > Assign Users page.
- You should see 2 new links at the top of the table: `All` and `Unassigned`. Both should display the total of users and the total number of unassigned users within the parenthesis of each filter.
- Click on the `Unassigned` filter. Only the unassigned users should be displayed in the table.
- Click on the `All` filter. All the users should be displayed in the table, regarding whether they are assigned or not to an institution.
- Repeat the process on the Institutions > Assign Books page.

### CODE REVIEW

File: resources/views/assign/index.blade.php [hunk 1]
```diff
@@ -5,20 +5,22 @@
 @endif
 
 <div class="wrap">
-	<h1 class="wp-heading-inline">{{ __('Assign Users', 'pressbooks-multi-institution') }}</h1>
+	<h1 class="wp-heading-inline">{{ $title }}</h1>
```
Comment (richard015ar) [inline resources/views/assign/index.blade.php:8]: Both `books/assign.blade.php` and `users/assign.blade.php` were the same file, except for 2 things:
- Title page
- Form ID : `pressbooks-multi-institution-assign-users` and `pressbooks-multi-institution-assign-books`
Both referenced in https://github.com/pressbooks/pressbooks-multi-institution/pull/109/files#diff-0d219c0f839b726aa2667e9110ffde0759fa3d912c94644e09f4dfceba3c52adL60

**Review by SteelWagstaff — COMMENTED** [review]
I like what I see here -- nice work, Ricardo!

File: resources/views/assign/index.blade.php (hunk 1 above)
Comment (SteelWagstaff) [reply resources/views/assign/index.blade.php:8]: Good improvement -- thanks for taking the time to simplify things!

**Review by arzola — APPROVED** [review]
LGTM but there are some merge conflicts

**Pull request merged**